configure.ac

```diff
@@ -216,6 +216,20 @@ AC_CHECK_FILE([$srcdir/extern/fmt-5.3.0/README.rst],
                [], 
                []
                [AC_MSG_ERROR([fmt-5.3.0 is not available, please download fmt-5.3.0 from https://fmt.dev/ and put the unpacked archive in the libsemigroups/extern directory])])
+
+# From https://stackoverflow.com/q/23379805/
+
+AC_CANONICAL_HOST
+X86_CXXFLAGS=""
+AS_CASE([$host_cpu],
+        [x86_64],
+        [
+         X86_CXXFLAGS="-mavx -march=native"
```

Comment:
Is it possible to add a configure option to disable this as well? For binary packages like conda packages `-march=native -mavx` are harmful as the built binaries can only be run on a newer architecture than the build machine.

Reply:
Sure thing, I'll add this to the PR now too

Reply:
Any name suggestions?

Reply:
NTL uses `--disable-native`. Btw, `-mavx -march=native` can be replaced by just `-march=native`

Reply:
Thanks, I'm working on this now.

Reply:
One more question @isuruf: should I account for the case where `-march=native` is not supported but `-mavx` is? If the answer is: no, then is the configure option `--disable-native` really required? The  setup I'm working on would not include the flag `-march=native` unless libsemigroups is compiled with `--enable-hpcombi`, so the flag (and HPCombi support) can be disabled by passing `--disable-hpcombi`. Thanks again

Reply:
Or maybe it'd be better to say: If `-march=native` is not supported , then with the changes in the current PR the flag won't be added.